guide/ops/externalized-configuration.md

```diff
@@ -64,6 +68,18 @@ brooklyn.external.credentials.appId=MyApp
 
 In this case, the list of servers is stored in a properties file located on an Operations Department web server, and the
 credentials are stored in an instance of [Vault](https://www.vaultproject.io/).
+More information on these providers is below.
+
+For demo purposes, there is a pre-defined external provider called
+`brooklyn-demo-sample` which defines `hidden-brooklyn-password` as `br00k11n`.
+This is used in some of the sample blueprints, referencing `$brooklyn:external("brooklyn-demo-sample", "hidden-brooklyn-password")`. 
+The value used here can be overridden with the following in your `brooklyn.properties`:
```

Comment:
`brooklyn.cfg` as the next release will be karaf?

Reply:
that will need to be changed in lots of places, including the first para of this doc.  changing just here is weird.  suggest it be a different tranch of work.

Reply:
They have been changed in #210 already. I'll merge this though and make a new PR.